vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md

```diff
@@ -110,13 +111,16 @@ We recommend setting up a test application so you can easily verify the ingress
 
 ### Install the Entando Custom Resource Definitions (CRDs)
 Once per cluster you need to deploy the `Entando Custom Resources`.
-
-1. Download the Custom Resource Definitions (CRDs) and deploy them
+1. Create the namespace
+```
+kubectl create namespace entando
+```
+2. Download the Custom Resource Definitions (CRDs) and deploy the cluster scoped resources
 ```
-kubectl apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/cluster-resources.yaml
+kubectl apply -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/cluster-resources.yaml
```

Comment:
Should we keep the namespace parameter here or the "create namespace" previous command put it as the default one?

Reply:
Just to confirm what we discussed on slack, the namespace isn't required for a cluster-level operation.